ui/tutorial-renderer/src/main/res/layout/tutorial_tips_learn.xml

```diff
@@ -20,24 +20,25 @@
 
         <include layout="@layout/tutorial_tips_include_positioning" />
 
-        <ImageView
-            android:id="@+id/image"
+        <com.airbnb.lottie.LottieAnimationView
+            android:id="@+id/animation"
             android:layout_width="0dp"
             android:layout_height="0dp"
-            android:scaleType="fitCenter"
-            android:src="@drawable/img_tutorial_tips_people"
             app:layout_constraintBottom_toBottomOf="@id/imagePosition"
             app:layout_constraintEnd_toEndOf="@id/imagePosition"
             app:layout_constraintStart_toStartOf="@id/imagePosition"
-            app:layout_constraintTop_toTopOf="@id/imagePosition" />
+            app:layout_constraintTop_toTopOf="@id/imagePosition"
+            app:lottie_autoPlay="true"
+            app:lottie_rawRes="@raw/anim_tutorial_tips_people"
+            app:lottie_loop="true" />
 
         <TextView
             android:id="@+id/headline"
             android:layout_width="0dp"
             android:layout_height="wrap_content"
             android:breakStrategy="balanced"
             android:gravity="center_horizontal"
-            android:text="@{@string/tutorial_tips_learn_headline(formatArgs.getString(TutorialActivity.FORMAT_ARG_TOOL_NAME)), default=@string/tutorial_tips_learn_headline}"
+            android:text="@string/tutorial_tips_learn_headline"
```

Comment:
Was this dynamically inserting the tool name into the text before but we decided to go with ```this tool``` instead?

Reply:
yeah